carmin-server/server/resources/path.py

```diff
@@ -1,19 +1,24 @@
 import os
```

Comment:
Please try to extract the non-endpoint functions outside of this resource file. I believe it would help to make this resource simpler to read and understand which functions are actions and which are helper functions.